ci-operator/config/openshift/splunk-forwarder-operator/openshift-splunk-forwarder-operator-master.yaml

```diff
@@ -34,6 +38,42 @@ tests:
     make verify
   container:
     from: src
+- as: vuln-check-operator
+  commands: |
+    cd /tmp
```

Comment:
The one question I have, is would these commands be better placed in a script/Makefile in the repo itself, rather than here?

Reply:
The first iteration of this was running a make target from the src container, but as I need to run commands inside the built containers, that didn't work.

Also, in these image containers, I didn't see a way of referencing a script from the "src" container (where a script might live, like in the "verify" test). What do you think about curl-ing one from the repo Github?

Reply:
This PR as is, is better than curling a script into the container IMO.

Reply:
Soo, can I have some :bacon: ?